### ISSUE TITLE
CrIS-FSR: missing check on skin temperature sensitivity over non-water surfaces 

### ISSUE THREAD

**Issue by rtodling** [GEOS-ESM/swell#472]
I believe the yaml files for CrIS-FSR are not taking into account changes made to GSI to adjust errors over non-water surfaces, that is, the following:

https://github.com/GEOS-ESM/GEOSana_GridComp/blob/495e2f0772bac4b258e6cd556966f73638bc86e4/GEOSaana_GridComp/GSI_GridComp/qcmod.f90#L2463-L2475

doesn't seem to have an equivalent in the yaml settings in SWELL.

**Comment by gmao-wgu** [comment]
That is because of the current configuration used in GEOS, where CrIS-FSR observations are only assimilated over the ocean. Therefore, this QC is not applicable.

**Comment by rtodling** [comment]
> That is because of the current configuration used in GEOS, where CrIS-FSR observations are only assimilated over the ocean. Therefore, this QC is not applicable.

Yes, I understand, but I wonder if it would be safer to add the check for consistency w/ GSI in case we decide to use the channels over non-water surfaces. If you don't think so ... then we can drop it.

**Comment by gmao-wgu** [comment]
If you don’t mind, I would prefer to work on this after we decide to assimilate CrIS-FSR over non-water surfaces in JEDI.
